### PR TITLE
chef 12 windows 2012r2

### PR DESCRIPTION
**Pull request by heathsnow**
- [ ] @laynes
- [x] @sweitzel74

* chef 12
* windows 2012r2
* rubocop

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-source 'https://supermarket.chef.io'
+source 'http://berkshelfapi.daptiv.com'
```
Comment (sweitzel74) [inline Berksfile]: Public repo needs external urls

File: Gemfile [hunk 2]
```diff
@@ -1,14 +1,13 @@
-source 'https://rubygems.org'
+source 'http://artrepo.daptiv.com:8081/artifactory/api/gems/rubygems-virtual/'
```
Comment (sweitzel74) [inline Gemfile]: Public repo needs external urls

File: Gemfile.lock [hunk 3]
```diff
@@ -1,7 +1,10 @@
 GEM
-  remote: https://rubygems.org/
+  remote: http://artrepo.daptiv.com:8081/artifactory/api/gems/rubygems-virtual/
```
Comment (sweitzel74) [inline Gemfile.lock]: Public repo needs external urls

File: .kitchen.yml [hunk 4]
```diff
@@ -7,9 +7,9 @@ provisioner:
   name: chef_zero
 
 platforms:
-  - name: windows2012r2
+  - name: windows2012r2_chef12
     driver:
-      box: daptiv/windows2012r2_chef11
+      box_url: 'http://artrepo.daptiv.com:8081/artifactory/vagrant-local/virtualbox_windows-2012r2_chef-12.5.1_1.0.23.box'
```
Comment (sweitzel74) [inline .kitchen.yml]: Public repo needs external urls

**Comment by heathsnow** [comment]
Please take another look @sweitzel74

File: metadata.rb [hunk 5]
```diff
@@ -1,10 +1,10 @@
-name             'fiddler'
-maintainer       'Daptiv Solutions, LLC'
+name 'fiddler'
+maintainer 'Daptiv Solutions, LLC'
```
Comment (sweitzel74) [inline metadata.rb]: Need to update in here

**Comment by sweitzel74** [comment]
LGMT :shipit:

**Pull request merged**